sklearn/decomposition/dict_learning.py

```diff
@@ -304,7 +321,7 @@ def sparse_encode(X, dictionary, gram=None, cov=None, algorithm='lasso_lars',
 
 
 def _update_dict(dictionary, Y, code, verbose=False, return_r2=False,
-                 random_state=None):
+                 random_state=None, dict_positive=False):
```

Comment:
I had thought about this, but was concerned it might be unclear. If you would prefer it, I can make the change.